spec/requests/security_activity_spec.rb

```diff
@@ -210,6 +211,19 @@
       expect_event SecurityActivity::PASSWORD_CHANGED
       expect_event_on_security_page SecurityActivity::PASSWORD_CHANGED
     end
+
+    it "record a USER_DESTROYED event" do
+      attribute_service_url = "example.gov.uk"
+
+      ClimateControl.modify ATTRIBUTE_SERVICE_URL: attribute_service_url do
+        allow(Rails.logger).to receive(:public_send)
```

Comment:
Could we have some check on what is `public_send`ed?  Something that would have caught the fact that in the code above it's currently doing `Rails.logger.public_send Rails.application.config.log_level, nil`